cairo1-run/src/main.rs

```diff
@@ -180,7 +180,10 @@ fn run(args: impl Iterator<Item = String>) -> Result<Option<String>, Error> {
                 .build()
                 .unwrap();
             let main_crate_ids = setup_project(&mut db, &args.filename).unwrap();
-            compile_prepared_db(&mut db, main_crate_ids, compiler_config).unwrap()
+            let sierra_program_with_dbg =
```

Comment:
Why is it called `with_dbg`?

Reply:
Because its is `SierraProgramWithDebug`.

Reply:
Nice